src/proxy-ui-api/frontend/src/views/KeysAndCertificates/SignAndAuthKeys/UnknownKeysTable.vue

```diff
@@ -94,9 +94,9 @@ export default Vue.extend({
   },
   computed: {
     hasPermission(): boolean {
```

Comment:
Would method name `canCreateCsr` be more descriptive (and same style seems to be used elsewhere)?

Reply:
Fixed